schemas/NDAr_NDArray_schema.fbs

```diff
@@ -1,5 +1,4 @@
-//This schema is under development.
-//If you decide to use it, be aware that backwards compatibility is not guaranteed.
+// This schema is not in active use.
```

Comment:
I think this is misleading, there is a file writer module for this, and we will be trying the AD plugin at ISIS in the very near future.